.github/workflows/publish.yml

```diff
@@ -10,26 +10,4 @@ on:
 jobs:
   publish-artifacts:
     name: Publish / Artifacts
-    runs-on: ubuntu-20.04
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-        with:
-          # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
-          fetch-depth: 0
-
-      - name: Set up JDK 8
-        uses: olafurpg/setup-scala@v13
-        with:
-          java-version: adopt@1.8
-
-      - name: Cache Coursier cache
-        uses: coursier/cache-action@v6.2
-
-      - name: Publish artifcats
-        run: sbt ci-release
-        env:
-          PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
-          PGP_SECRET: ${{ secrets.PGP_SECRET }}
-          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
-          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+    uses: ihostage/.github/.github/workflows/publish.yml@main
```

Comment:
🤔 https://github.community/t/using-organization-secrets-in-reusable-workflows/205207
Need to try without passing secrets explicitly. If it won't work then we'll change it.

Reply:
Nope, it didn't work 😞 Need to set secrets explicitly. Hurtful 😞 
https://github.com/playframework/play-json/runs/4985224723?check_suite_focus=true